nf_core/pipeline-template/{{cookiecutter.name_noslash}}/nextflow.config

```diff
@@ -75,13 +75,12 @@ dag {
 }
 
 manifest {
-  name = '{{ cookiecutter.name }}'
```

Comment:
I think you can probably leave this here. Although it's not supported yet, it doesn't do any harm and it will hopefully be added soon (see https://github.com/nextflow-io/nextflow/issues/871).

If we remove it, we also have to update the tests and linting, then we'll need to add that back again once it's added to nextflow.

Reply:
Ok. Ive made a couple more changes in the linting code where required. How can I run the tests locally?

Reply:
Nice! To run locally you can just run the same commands that travis does: https://github.com/nf-core/tools/blob/8e675af22699cda846c12ee0f205a4e4f6796261/.travis.yml#L24-L26

So there are the python tests which you can do with `python -m pytest .` (no need for coverage analysis). Then the other test is to create and then lint a pipeline.

Reply:
Oh, you may need to install some extra dependencies for the python testing first:

```
 pip install pytest pytest-datafiles mock
```